src/olympia/reviewers/models.py

```diff
@@ -1197,19 +1209,3 @@ class Meta:
     def __unicode__(self):
         return u'[%s] private: |%s| public: |%s|' % (
             self.addon.name, self.private, self.public)
-
-
-@Whiteboard.on_change
-def watch_public_whiteboard_changes(old_attr=None, new_attr=None,
```

Comment:
why is this being removed?  I didn't see a mention of it.

Reply:
The way I understood the issue, you need to use the new "Clear request for information" action, which is behind a `Reviews:Admin` check, to, well, clear a request for information now. So this bit looked obsolete.

Reply:
Isn't this also called for developer changes to the Whiteboard though?  (The model is in `/reviewers/models.py` but it's used in devhub too, iirc)

Reply:
Right, but developers now have an explicit checkbox when submitting/editing a version to clear the info request.

Reply:
Must have forgotten that part - it was a long patch!  Nothing to see here then.